autoload/go/doc.vim

```diff
@@ -85,21 +85,32 @@ function! go#doc#Open(newmode, mode, ...)
     endif
 
     let command = printf("%s %s", bin_path, join(a:000, ' '))
+    let out = go#util#System(command)
   else
     " check if we have 'gogetdoc' and use it automatically
     let bin_path = go#path#CheckBinPath('gogetdoc')
     if empty(bin_path)
       return
     endif
 
+
     let offset = go#util#OffsetCursor()
     let fname = expand("%:p:gs!\\!/!")
     let pos = shellescape(fname.':#'.offset)
-
     let command = printf("%s -pos %s", bin_path, pos)
+    let in = ""
+
+    if &modified
```

Comment:
Can you add a comment above this variable to indicate the `gogetdoc` has a feature that we can make use of